bundles/org.openhab.binding.mqtt.fpp/src/main/java/org/openhab/binding/mqtt/fpp/internal/handler/FPPPlayerHandler.java

```diff
@@ -0,0 +1,280 @@
+                    executeGet("/api/playlist/" + lastPlaylist + "/start");
+                }
+            } else if (command == NextPreviousType.NEXT) {
+                executeGet("/api/command/Next Playlist Item");
+            } else if (command == NextPreviousType.PREVIOUS) {
+                executeGet("/api/command/Prev Playlist Item");
+            }
+        }
+        if (channelId.equals(CHANNEL_VOLUME)) {
+            Integer volume = null;
+            if (command instanceof PercentType percentCommand) {
+                volume = percentCommand.intValue();
+            } else if (command == OnOffType.OFF) {
+                volume = 0;
+            } else if (command == OnOffType.ON) {
+                volume = lastKnownVolume;
+            } else if (command == IncreaseDecreaseType.INCREASE) {
+                if (lastKnownVolume < 100) {
+                    lastKnownVolume++;
+                    volume = lastKnownVolume;
+                }
+            } else if (command == IncreaseDecreaseType.DECREASE) {
+                if (lastKnownVolume > 0) {
+                    lastKnownVolume--;
+                    volume = lastKnownVolume;
+                }
+            }
+            if (volume != null) {
+                lastKnownVolume = volume;
+                executePost("/api/system/volume", "{\"volume\":" + lastKnownVolume + "}");
+                updateState(CHANNEL_VOLUME, new PercentType(lastKnownVolume));
+            }
+        }
+        if (channelId.equals(CHANNEL_TESTING)) {
+            if (command == OnOffType.OFF) {
+                executePost("/api/testmode", GetTestMode(0));
+            } else if (command == OnOffType.ON) {
+                executePost("/api/testmode", GetTestMode(1));
+            }
+        }
+    }
+
+    private String GetTestMode(int enable) {
+        return "{\"mode\":\"RGBChase\",\"subMode\":\"RGBChase-RGB\",\"cycleMS\":1000,\"colorPattern\":\"FF000000FF000000FF\",\"enabled\":"
+                + enable + ",\"channelSet\": \"1-1048576\",\"channelSetType\": \"channelRange\"}";
+    }
+
+    @Override
+    public void initialize() {
+        config = getConfigAs(ConfigOptions.class);
+
+        fullStatusTopic = config.playerMQTT + "/" + STATUS_TOPIC;
+        fullVersionTopic = config.playerMQTT + "/" + VERSION_TOPIC;
+        bridgeStatusChanged(getBridgeStatus());
+    }
+
+    @Override
+    public void processMessage(String topic, byte[] payload) {
+        String state = new String(payload, StandardCharsets.UTF_8);
+        logger.trace("Received the following new FPP state:{}:{}", topic, state);
+
+        if (topic.endsWith(STATUS_TOPIC)) {
+            processIncomingState(state);
+        } else if (topic.endsWith(VERSION_TOPIC)) {
+            updateState(CHANNEL_VERSION, new StringType(state));
+            updateStatus(ThingStatus.ONLINE);
+        }
+    }
+
+    public ThingStatusInfo getBridgeStatus() {
+        Bridge b = getBridge();
```

Comment:
Use meaningfull names.
```suggestion
        Bridge bridge = getBridge();
```